### ISSUE TITLE
Laud 108 images?

### ISSUE THREAD

**Issue by mmwwah** [scottkleinman/aeme#69]
I've been using Mary of Egypt as an analogue (although I know that they're all usable for that purpose now, or so I gather).  But what I don't know is the location of the images, so that I can compare encoding with what is on the ms. page.  Are those available? 

As a separate question, what is the status of the Junius 1 images?  Last I checked, the first nine folia were viewable.  Both Zach and I have been using the b&w scans that Rani made last summer, which is all we need 99.5% of the time.

**Comment by scottkleinman** [comment]
I am trying to get an image server up and running, but it is taking a long time. It may be that the temporary viewer has up to 20 images now, but I think I may have run out of memory (and the viewer was clunky). So, for now, the best bet is to search for the images in [CSUN ScholarWorks](http://scholarworks.csun.edu/handle/10211.3/138539). In some cases, a JPEG 2000 viewer has been generated automatically, but this is a bit erratic. If there's no JPEG 2000 link, you'll have to download the TIFF image. If necessary, I can generate a bunch of lower quality JPEGs for you. Experience tells me that you have to reduce the quality quite a lot before you even notice a difference.

**Comment by mmwwah** [comment]
No, we're cool on images for Junius 1.  My main concern was finding an image for Mary of Egypt.

And if there's another section or page that would be a more instructive exemplar for me, don't hesitate to recommend.

**Comment by scottkleinman** [comment]
Oh, for Laud Misc. 108 images, the best place is [http://tsar192.grid.csun.edu/osdviewer.php](http://tsar192.grid.csun.edu/osdviewer.php), fol. 117v for Mary of Egypt.

**Comment by mmwwah** [comment]
Thanks -- I found it via Scholarworks and the folio number in the tei-header.

But it raises another question (don't they all?):  The first image in our Junius 1 images is the cover of the codex.  Whereas 1ar of Laud 108 appears to be text.  Is that going to matter to us?  Will we encode the few words on the cover?  And the alphabet on the flyleaf that is considered folio 1 (following Francis Junius's numbering, which I intend to keep)?

I'm asking because Zach has assumed that the first page of Ormulum text is folio 1, so I need to go back and renumber.  And I'd rather measure twice and cut once.

**Comment by scottkleinman** [comment]
The Laud Misc. 108 numbering is taken from whatever the Bodleian gave us and will probably be changed. You can do the same with Junius 1, where the situation is even more sketchy. In ScholarWorks, the folios are numbered in the order I got them, but that was meant to be temporary. I will change the numbers to whatever you come up with.

**Comment by mmwwah** [comment]
Okay.  My goal is to retain FJ's numbering, because it makes everything easier.  But that means that image 5 is folio 3.  Maybe this is something to deal with later?

**Comment by scottkleinman** [comment]
Yes, encode `<pb>` however you want the foliation to be represented. You can include something like this:

```xml
<pb n="5"> <!-- bxa0003 -->
```
That way, we'll know which image file is used, and we can change the image metadata later.

**Comment by mmwwah** [comment]
So `<pb>` should reflect foliation and not image file number, right?  That's what you said, but the example has the numbers reversed, so I want to make sure I've got the right end of the stick.

**Comment by scottkleinman** [comment]
I was in the process of getting kicked out of Starbucks because they "needed the table to set up their holiday merchandise". That should have been `<pb n="3"> <!-- bxa0005 -->`.
